pubspec.yaml

```diff
@@ -16,8 +16,8 @@ dev_dependencies:
     sdk: flutter
 
 environment:
-  sdk: '>=2.17.0 <3.0.0'
-  flutter: ">=3.4.0-34.1.pre"
```

Comment:
@dnfield @jonahwilliams I know this has been merged already, but is there a reason why this was raised? I personally don't want to upgrade to 3.7 yet because of:
1. Some pretty huge UI glitches that I saw when I tested my app on it (waiting on flutter team to fix those before I upgrade)
2. Changes in material components that I don't want by default (so I'll need to spend some time reviewing everything from my app)

I expect I'm probably not the only one postponing the migration to the latest flutter version.

Reply:
Ah, seeing that this merge did not change any dependencies I thought there's none requiring new features from flutter 3.7.

In case I was not clear, the glitches are unrelated to this library.

Reply:
Great! But I also work on visual glitches in the framework/engine :)

Reply:
I'll check just now to see if 3.7.1 still has them. I did not get a chance to file a bug report because I'm pre-launch for the app that I built over the last year. Pretty much decided to postpone any 3.7 investigations and bug reporting until the app is live (+ did not want to run the risk of new bugs just before going live)

Reply:
@dnfield seems I was wrong. Not all the issues are gone. Just created an issue in flutter's github: https://github.com/flutter/flutter/issues/119965